frameworks/template/build.self.hosted.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+# capture anonymous metrics for reporting
+curl https://mesosphere.com/wp-content/themes/mesosphere/library/images/assets/sdk/build-sh-start.png >/dev/null 2>&1
+
+FRAMEWORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+BUILD_DIR=$FRAMEWORK_DIR/build/distributions
+PUBLISH_STEP=${1-none}
+export REPO_NAME=template
```

Comment:
Should `REPO_NAME` just be the name of the current directory? `basename $FRAMEWORK_DIR`

I don't know the answer to this, to be clear